File: examples/stock/quote/node/getAStockQuote.js
import * as stock from "@m3o/services/stock";

// Get the last quote for the stock
async function GetAstockQuote() {
  let stockService = new stock.StockService(process.env.MICRO_API_TOKEN);
  let rsp = await stockService.quote({
    symbol: "AAPL",
  });
  console.log(rsp);
}

await GetAstockQuote();

File: examples/stock/quote/go/getAStockQuote.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/stock"
	"os"
)

// Get the last quote for the stock
func GetAstockQuote() {
	stockService := stock.NewStockService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := stockService.Quote(&stock.QuoteRequest{
		Symbol: "AAPL",
	})
	fmt.Println(rsp, err)
}
